numpy/lib/tests/test_stride_tricks.py

```diff
@@ -274,7 +275,9 @@ def test_broadcast_to_raises():
 
 
 def test_broadcast_shape():
-    # broadcast_shape is already exercized indirectly by broadcast_arrays
+    # tests internal _broadcast_shape
+    # _broadcast_shape is already exercized indirectly by broadcast_arrays
+    # _broadcast_shape is also exercized by the public broadcast_shapes function
```

Comment:
```suggestion
    # _broadcast_shape is already exercised indirectly by broadcast_arrays
    # _broadcast_shape is also exercised by the public broadcast_shapes function
```
(unless that is a common British spelling, in which case it is fine).

Reply:
:) The original comment that was in there (regarding `broadcast_arrays`) used "ize" and I assumed it was British English, but a Google search indicates that exercise is spelled with "ise" on both sides of the pond.